homework4-2.py

```diff
@@ -0,0 +1,21 @@
+import copy
+# Use a list comprehension to construct the list
+# ['ab', 'ac', 'ad', 'bb', 'bc', 'bd']
+lst = [chr(97 + i) + chr(98 + j) for i in range(2) for j in range(3)]
```

Comment:
characters numbers looks too complex. i.e. for 'a' and 'b' we can use `'ab'` or `['a', 'b']` (I like first one more but they are equal). And so on for other same issues. It's more native to use these constructions instead of range. Range is rare used functionality to be honest